src/main/java/duke/Task.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+public class Task {
```

Comment:
Perhaps an alternative way this class is defined could be as an abstract class since `Tasks` are always `Todo`, `Deadline` or `Event`?